src/jit/codegenxarch.cpp

```diff
@@ -8027,6 +8059,19 @@ CodeGen::genPutStructArgStk(GenTreePtr treeNode, unsigned baseVarNum)
     assert(baseVarNum != BAD_VAR_NUM);
     
     var_types targetType = treeNode->TypeGet();
+
```

Comment:
I would suggest to abstract the codegen routines that pass SIMD args on stack into separate routines (e.g. GenPutSIMDArgStk()).This would allow us in future to scale to other arch/targets/ABI.